### PR TITLE
Update main.py added /arrest

### PR DESCRIPTION
**Pull request by qwe123coder**
i forgot to remove arrest pls remove it

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import discord
-from discord_slash import SlashCommand
+from discord_import SlashCommand
```
Comment (tyffical) [inline main.py:2]: Again, don't change this line

File: main.py [hunk 2]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
...
+          name="reason",
+          description="Why are you arrest blahajganger ?",
+          option_type=3, #corresponds to STRING
+          required=False)
+          ])
+async def arrest(ctx, recipient=None, reason=None):
+    mention = recipient.id if recipient else None
+    myid = ctx.author_id 
+    if not mention:
+        await ctx.send(content="To whom should I arrest?")
```
Comment (tyffical) [inline main.py:123]: Should be "Whom should I arrest?"

File: main.py [hunk 3]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
...
+            content="<@{mention}>, you got arrested \n reason: {reason}".format(
+                mention=mention, reason=reason))
+
+   
+              
+    elif mention == myid:
+        await ctx.send(content="Ha! you can't arrest yourself.")
+    else:
+        await ctx.send(
+            content="<@{mention}>, you got arrested \n reason: {reason}".format(
```
Comment (tyffical) [inline main.py:137]: Should be "You're under arrest!"

Comment (tyffical) [inline main.py:137]: Use the same if if elif else structure that the old arrest command used.

File: main.py [hunk 4]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
...
 
+@slash.slash(name="arrest", guild_ids=guild_ids_list, description="for some reason blahajgangers wanted to arrest one another?", 
+options=[create_option(
+          name="recipient",
+          description="Who do you want to arrest ?",
+          option_type=6, #corresponds to USER
+          required=False),
+        create_option(
+          name="reason",
+          description="Why are you arrest blahajganger ?",
```
Comment (tyffical) [inline main.py:115]: Should be "Why should they be arrested?"

**Review by tyffical — CHANGES_REQUESTED** [review]
As I told you on Discord, please resolve all the grammatical errors.

File: main.py (hunk 3 above)
Comment (qwe123coder) [reply main.py]: @tyffical i used this as the one you had was giving some bugs so i mad my one it work just fine as the old one
the old code even when i gave whom and why 

old code 
image


new code 

image

File: main.py [hunk 5]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
                 mention=mention, reason=reason),
             file=discord.File('giftBlahaj.png'))
 
+@slash.slash(name="arrest", guild_ids=guild_ids_list, description="for some reason blahajgangers wanted to arrest one another?", 
+options=[create_option(
+          name="recipient",
+          description="Who do you want to arrest ?",
```
Comment (tyffical) [inline main.py:110]: Change to whom

**Review by tyffical — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (qwe123coder) [reply main.py]: > Use the same if if elif else structure that the old arrest command used.

it dose not work

File: main.py [hunk 6]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
...
+          required=False)
+          ])
+async def arrest(ctx, recipient=None, reason=None):
+    mention = recipient.id if recipient else None
+    myid = ctx.author_id 
+    if not mention:
+        await ctx.send(content="Whom should I arrest?")
+    
+    elif not reason:
+        reason = "yeet just for fun! :"
```
Comment (tyffical) [inline main.py:126]: Missing ) for smiley face :)

File: main.py [hunk 7]
```diff
@@ -103,7 +104,40 @@ async def gift(ctx, recipient=None, reason=None):
                 mention=mention, reason=reason),
             file=discord.File('giftBlahaj.png'))
 
+@slash.slash(name="arrest", guild_ids=guild_ids_list, description="for some reason blahajgangers wanted to arrest one another?", 
+options=[create_option(
+          name="recipient",
+          description="Whom do you want to arrest ?",
```
Comment (tyffical) [inline main.py:110]: Delete the weird space before the ?

**Review by tyffical — CHANGES_REQUESTED** [review]

File: main.py (hunk 6 above)
Comment (qwe123coder) [reply main.py]: fixed

File: main.py (hunk 7 above)
Comment (qwe123coder) [reply main.py]: fixed

**Review by tyffical — APPROVED** [review]

**Review by nishant0078 — COMMENTED** [review]
Screenshot from 2021-07-24 19-27-23


Please correct the formatting here

**Comment by qwe123coder** [comment]
@nishant0078 i did not get you can you tell what line i messed up

**Comment by nishant0078** [comment]
Look at the Image it is attached the indentation is wrong all over the place

**Comment by qwe123coder** [comment]
this 
image
? 
if yess it still works fine

**Comment by nishant0078** [comment]
Alright Code is good to go. Hope you have fixed all the grammar error

**Comment by tyffical** [comment]
It looks alright, but there are merge conflicts with main branch that need to be fixed

**Review by tyffical — CHANGES_REQUESTED** [review]
Pull from main branch and fix merge conflicts

**Pull request merged**